src/CloudfrontStack.ts

```diff
@@ -76,6 +111,7 @@ export class CloudfrontStack extends Stack {
       {
         viewerProtocolPolicy: ViewerProtocolPolicy.REDIRECT_TO_HTTPS,
         cachePolicy: CachePolicy.CACHING_DISABLED,
+        //responseHeadersPolicy: this.corsHeadersPolicy,
```

Comment:
Waarom staat deze uit?

Reply:
die regel had ik weg moeten halen; de default behaviour die alleen redirect naar de security.txt op Nijmegen.nl, daar is geen cors nodig.